pyvrp/GeneticAlgorithm.py

```diff
@@ -49,25 +76,10 @@ def __init__(
         crossover_op: CrossoverOperator,
         params: GeneticAlgorithmParams = GeneticAlgorithmParams(),
     ):
-        """
-        Creates a GeneticAlgorithm instance.
+        if len(population) < 2:
+            # Cannot do meaningful crossover without at least two individuals.
+            raise ValueError("Expected population size at least two.")
```

Comment:
Related: I agree with Population being empty as default is fine, since the Population is just a container (related discussion https://github.com/N-Wouda/PyVRP/pull/168#discussion_r1125512469). 

But then there's another issue/responsibility for GA. When restarting the Population (without any initial solutions) this will result in a population without individuals, so we can't do any crossovers after restart.

Reply:
>It's only when we don't have any individuals that there will be problems, since then there can be no crossover at all. So I think we should require at least one individual.

Works for me, I'll update this to "nonempty".

>But then there's another issue/responsibility for GA. When restarting the Population (without any initial solutions) this will result in a population without individuals, so we can't do any crossovers after restart.

I'm not sure who needs to fix this - the user, the GA, or the Population. I'm leaning user, but I also think there's a deeper issue with Population and its `params.min_pop_size` argument, for example. We haven't gotten that right yet.

Reply:
I have an idea about this; see Slack!